src/service/private.overseas.entity.details.ts

```diff
@@ -30,3 +30,32 @@ export const getPrivateOeDetails = async (
 
   return response.resource;
 };
+
+export const getManagingOfficerPrivateData = async (
```

Comment:
I'm assuming we can now update the version of the `api-sdk-node` in the `package.json`?